axon-server-connector/src/main/java/org/axonframework/axonserver/connector/command/AxonServerCommandBus.java

```diff
@@ -191,7 +202,7 @@ public Registration subscribe(String commandName, MessageHandler<? super Command
                              + "Expect similar logging on the local segment.", commandName);
         Registration localRegistration = localSegment.subscribe(commandName, messageHandler);
         io.axoniq.axonserver.connector.Registration serverRegistration =
-                axonServerConnectionManager.getConnection()
+                axonServerConnectionManager.getConnection(context)
```

Comment:
Just out of curiosity, but would you mind reminding me why we required the buses to be aware of the `context` they're getting the connection for instead of leaving that job for the `AxonServerConnectionManager`?
Was this because of this automatic addition/removal of tenant contexts perhaps?